pkgs/applications/science/logic/aspino/default.nix

```diff
@@ -23,7 +23,10 @@ stdenv.mkDerivation {
   postPatch = ''
     substituteInPlace Makefile \
       --replace "GCC = g++" "GCC = c++"
-
+    substituteInPlace src/main.cc \
+      --replace "defined(__linux__)" "defined(__linux__) && defined(__x86_64__)"
+    substituteInPlace src/MaxSatSolver.cc \
+      --replace "occ[i][sign(softLiterals[j])] > 0" "occ[i][sign(softLiterals[j])] != 0"
     patchShebangs .
```

Comment:
While we are at it: can we change this to a directory that needs patching?

Reply:
I didn't get this: you mean move it to a patch file?

Reply:
No, change dot to a few files or directories instead of everything

Reply:
I think we can even remove patchShebangs: #!/bin/sh seems perfectly legal to me.